Example/Stagehand Tutorial.playground/Pages/Repeating Animations.xcplaygroundpage/Contents.swift

```diff
@@ -9,14 +9,14 @@ import Stagehand
  By default, animations will run once before completing. Sometimes, though, we want our animation to loop through
  multiple times, sometimes even indefinitely.
 
- Using the `Animation.repeatStyle` property, we can control how our animation repeats.
+ Using the `Animation.implicitRepeatStyle` property, we can control how our animation repeats.
 
  */
 
 var animation = Animation<UIView>()
 
 // The default style is to not repeat.
-animation.repeatStyle = .none
+animation.Animation.implicitRepeatStyle = .none
```

Comment:
Oops, this is wrong. The playground doesn't build on CI, so didn't catch this. 😞